MEME/MEME/Artist/ArtistPortfolio/ArtistPortfolioEditingView/ArtistPortfolioEditingViewController.swift

```diff
@@ -78,19 +76,38 @@ class ArtistPortfolioEditingViewController: UIViewController, UINavigationContro
     
     //MARK: - setUI()
     private func setUI(){
+        makeupCategoryCollectionView.backgroundColor = .white
+        infoTextView.backgroundColor = .white
         self.tabBarController?.tabBar.isHidden = true
         infoTextView.delegate = self
         infoTextView.layer.cornerRadius = 10
         infoTextView.layer.borderWidth = 1
         infoTextView.layer.borderColor = UIColor.gray200.cgColor
+        firstImgView.layer.cornerRadius = 5
+        secondImgView.layer.cornerRadius = 5
+        thirdImgView.layer.cornerRadius = 5
+        
+        
+        firstDeleteButton.layer.cornerRadius = 5
+        firstDeleteButton.layer.maskedCorners = .layerMaxXMinYCorner
+        firstDeleteButton.layer.masksToBounds = true
+        
+        secondDeleteButton.layer.cornerRadius = 5
+        secondDeleteButton.layer.maskedCorners = .layerMaxXMinYCorner
+        secondDeleteButton.layer.masksToBounds = true
+        
+        thirdDeleteButton.layer.cornerRadius = 5
+        thirdDeleteButton.layer.maskedCorners = .layerMaxXMinYCorner
+        thirdDeleteButton.layer.masksToBounds = true
```

Comment:
```suggestion
        [firstDeleteButton, secondDeleteButton, thirdDeleteButton].forEach { button in
            button?.layer.cornerRadius = 5
            button?.layer.maskedCorners = .layerMaxXMinYCorner
            button?.layer.masksToBounds = true
        }
```
요롷게 for each를 이용해 한번에 작성하는 방법도 있으니 참고하시면 좋을 것 같습니당!